### DISCUSSION TITLE
GitHub Sponsors?

### DISCUSSION THREAD

**Discussion by connorlay** [nvim-treesitter/nvim-treesitter#1890] [Q&A]
First of all, big thanks to the core team for creating such a wonderful plugin ❤️ 

Are there any plans or interest in adding GitHub Sponsors as an additional way to fund this project? I know there is already an Open Collective setup, but I'm wondering if adding an additional funding source could help with exposure? For example, I have a list of badges on my GitHub profile that display the projects I am currently sponsoring. When people I follow start sponsoring a project, I see an update in my activity timeline.

What do folks think?

**Comment by theHamsta** [comment]
Thank you for this warm feedback! @vigoux knows more about sponsoring.

**Comment by vigoux** [comment]
Hey, I'll see what I can do with the Github sponsors, but from what I remember we made a mistake and were kind of stuck with GH sponsors, I'll get back to you with a link once I have one !
